src/decode-vxlan.c

```diff
@@ -268,9 +268,11 @@ static int DecodeVXLANtest02 (void)
 void DecodeVXLANRegisterTests(void)
 {
 #ifdef UNITTESTS
-    UtRegisterTest("DecodeVXLANtest01",
-                   DecodeVXLANtest01);
-    UtRegisterTest("DecodeVXLANtest02",
-                   DecodeVXLANtest02);
+	if (g_vxlan_enabled == true) {
```

Comment:
This is not how this is supposed to be handled. The tests should be registered unconditionally, and then the tests themselves can temporary update the global ```g_vxlan_enabled``` to true to run as normal. Be sure to reset them at the end.

Reply:
indent was off too it seems

Git commit message should be updated to something like "decoder/vxlan: disable by default"

Reply:
Oh OK.